src/Language/PureScript/Sugar/TypeClasses/Deriving.hs

```diff
@@ -121,13 +121,27 @@ deriveInstance mn syns _ ds (TypeInstanceDeclaration sa@(ss, _) ch idx nm deps c
            -> TypeInstanceDeclaration sa ch idx nm deps className tys . ExplicitInstance <$> deriveEq ss mn syns ds tyCon
            | otherwise -> throwError . errorMessage' ss $ ExpectedTypeConstructor className tys ty
       _ -> throwError . errorMessage' ss $ InvalidDerivedInstance className tys 1
+  | className == Qualified (Just dataEq) (ProperName "Eq1")
+  = case tys of
+      [ty] | Just (Qualified mn' _, _) <- unwrapTypeConstructor ty
+           , mn == fromMaybe mn mn'
```

Comment:
`maybe True (== mn) mn'` ?

I was a little confused there for a second :D There's also the more obscure `all (== mn) mn'` but I think that's too cute :D

Reply:
I just copied exactly the form of the other matches here. There's some other cleaning up I'd like to do actually, but didn't want to mix it in this PR :wink: so I can change it, but maybe it'd be better to keep them consistent for now?

Reply:
Should've looked at the whole file 🤦‍♂️